SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-3.0-or-later

pkg-config is not yet avaliable.

--- configure.ac	2021-04-21 17:02:44.864090500 +1000
+++ configure.ac	2021-04-21 17:03:31.468411937 +1000
@@ -160,19 +160,9 @@
 AC_FUNC_CLOSEDIR_VOID
 
 # See if the user wants to add (or not) GNU Guile support
-PKG_PROG_PKG_CONFIG
 AC_ARG_WITH([guile], [AS_HELP_STRING([--with-guile],
             [Support GNU Guile for embedded scripting])])
 
-# For some strange reason, at least on Ubuntu, each version of Guile
-# comes with it's own PC file so we have to specify them as individual
-# packages.  Ugh.
-AS_IF([test "x$with_guile" != xno],
-[ PKG_CHECK_MODULES([GUILE], [guile-2.0], [have_guile=yes],
-  [PKG_CHECK_MODULES([GUILE], [guile-1.8], [have_guile=yes],
-    [have_guile=no])])
-])
-
 AS_IF([test "$have_guile" = yes],
       [AC_DEFINE([HAVE_GUILE], [1], [Embed GNU Guile support])])
 
